packages/sources/coinapi/test/integration/assets.test.ts

```diff
@@ -8,86 +8,70 @@ describe('execute', () => {
   const execute = makeExecute()
   process.env.API_KEY = process.env.API_KEY ?? 'test_api_key'
 
-  describe('successful calls @integration', () => {
-    const requests = [
-      {
-        name: 'id not supplied',
-        testData: { data: { base: 'ETH', endpoint: 'assets' } },
-      },
-      {
-        name: 'from/to',
-        testData: { id: jobID, data: { from: 'ETH', endpoint: 'assets' } },
-      },
-      {
-        name: 'coin/market',
-        testData: { id: jobID, data: { coin: 'ETH', endpoint: 'assets' } },
-      },
-    ]
-
-    requests.forEach((req) => {
-      it(`${req.name}`, async () => {
-        const data = await execute(req.testData as AdapterRequest)
-        assertSuccess({ expected: 200, actual: data.statusCode }, data, jobID)
-        expect(data.result).toBeGreaterThan(0)
-        expect(data.data.result).toBeGreaterThan(0)
-      })
+  describe.each([
```

Comment:
Is this actually supported by jest? seems like its only supported for `it.each`

Reply:
https://jestjs.io/docs/api#describeeachtablename-fn-timeout